src/js/firebaseStorage.js

```diff
@@ -0,0 +1,95 @@
+// file : file, string : userId, string: type
```

Comment:
Did you mean
```suggestion
// string : file, string : userId, string: type
```

Reply:
its actually a file object that is passed into the function